frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -10,6 +10,8 @@ import {PackageManifestModel} from '../../models';
 import CatalogTileView from 'patternfly-react-extensions/dist/esm/components/CatalogTileView/CatalogTileView';
 import CatalogTile from 'patternfly-react-extensions/dist/esm/components/CatalogTile/CatalogTile';
```

Comment:
Not new, but these items should move up with other yarn imports. The yarn imports and console imports are grouped separately.